### PR TITLE
5386 Use form on selection page and delay on price update

### PR DESCRIPTION
**Pull request by RemcoSimonides**
Fix issue #5386

Updating the page is smooth as butter now :)

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/title/selection/selection.component.html [hunk 1]
```diff
@@ -15,69 +15,71 @@ <h1>My Selection</h1>
...
+                    alt="Movie Illustration" [delay]="0">
+                  <div fxLayout="column">
+                    <h3>{{ title.directors | displayName }}</h3>
+                    <p>{{ title | movieFeature}}</p>
+                  </div>
+                </article>
+              </section>
+              <mat-form-field appearance="outline">
+                <mat-label>Price</mat-label>
+                <input matInput type="number" min="0" [formControl]="contract.price" (input)="debouncedUpdatePriceControl(i, $event.target.value)"/>
```
Comment (fritzschoff) [inline apps/catalog/catalog/src/app/marketplace/title/selection/selection.component.html:48]: do you validate that the control gets the value in the debounced time?

Comment (fritzschoff) [reply apps/catalog/catalog/src/app/marketplace/title/selection/selection.component.html:48]: cause you bind the formControl to the html node

File: apps/catalog/catalog/src/app/marketplace/title/selection/selection.component.ts [hunk 2]
```diff
@@ -36,7 +40,13 @@ export class MarketplaceSelectionComponent {
     private snackBar: MatSnackBar
   ) {
     this.bucket$ = this.bucketQuery.selectActive().pipe(
-      tap(bucket => this.setTitle(bucket?.contracts.length)),
+      tap(bucket => {
+        if (!!bucket) {
```
Comment (fritzschoff) [inline apps/catalog/catalog/src/app/marketplace/title/selection/selection.component.ts:44]: use the filter operator for that

**Review by fritzschoff — APPROVED** [review]

**Comment by RemcoSimonides** [comment]
Form was barely used so I close this one in favor of https://github.com/blockframes/blockframes/pull/5429 in which the way prices are updated is changed. Total price is not updated instantly, but it is after 1 second.

**Pull request closed without merging**